modules/ROOT/pages/_partials/data-sync/js-client-offline.adoc

```diff
@@ -156,7 +169,7 @@ let config = {
 }
 ----
 
-=== Online Only Queries
+== Performing queries while online only
```

Comment:
```suggestion
== Ensuring queries or mutations are online only
```
Not sure about this one. The 'online only' here is a state we force on mutations so I'm not sure what you have sounds 100%.  What we really do is ensure certain mutations never get stored in our offline store.

Reply:
ok, let's have a chat about it tomorrow. I'm kinda puzzled how a developer accesses the 'mutation store':
1. I presume 'mutation store' = 'offline store'
2. If it's automatic, do we need to mention/name the store?

Reply:
1. The offline store is available directly on the client. So `client.offlineStore` will give them access to 4 functions allowing them to interact with their store.
2. The default process is that it is automatic - but if they mark their mutation with `@onlineOnly` then they are telling the SDK to _not_ persist this mutation in their offline store.

Reply:
@StephenCoady Spoke to Chris, for the heading how about one of the following:

* Excluding mutations from offline functionality
* Ensuring specified mutations are performed online only

Reply:
@finp I'd be +1 on the second option.